boards/arm/nucleo_l412rb_p/nucleo_l412rb_p.dts

```diff
@@ -53,14 +53,14 @@
 	current-speed = <115200>;
 };
 
-&spi1 {
-	pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5
-			 &spi1_miso_pa6 &spi1_mosi_pa7>;
+&spi2 {
+	pinctrl-0 = <&spi2_nss_pb12 &spi2_sck_pb13
+			 &spi2_miso_pb14 &spi2_mosi_pb15>;
```

Comment:
It might be confusing as the nucleo_412rb_p (mb1319) schematic rev C-01 set the PA11 as the SPI_CS and PB12 is not available on the Arduino connector

Reply:
If there's a conflict on revision, please at least add revision information in the board description.
Optionally, you can add multiple revisions support as this is on going [here](https://github.com/zephyrproject-rtos/zephyr/pull/34181)